src/mixins/common-mixin.ts

```diff
@@ -15,8 +15,8 @@ export default class CommonMixin extends Vue {
   /**
    * Removes the specified properties from nested objects.
    * @param baseObj the base object
-   * @param keys    the nested object keys which to omit properties from
-   * @param prop    the properties to be removed
+   * @param keys the nested object keys which to omit properties from
+   * @param prop the properties to be removed
```

Comment:
It's still formatted correctly:

![image](https://user-images.githubusercontent.com/10002889/222831373-43cd4648-9dc4-4add-968f-71197883864a.png)